openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -524,7 +524,7 @@ def enumerate_stereoisomers(
 
         """
         from rdkit import Chem
-        from rdkit.Chem.EnumerateStereoisomers import (
+        from rdkit.Chem.EnumerateStereoisomers import (  # type: ignore[import]
```

Comment:
A comment like this could be removed and instead moved into the config file. Neither are great solutions but putting it in the config file makes it re-usable if new additions to the toolkit also pull from this submodule.